protocol/tx/header.go

```diff
@@ -10,19 +10,31 @@ type header struct {
 		MinTimeMS, MaxTimeMS uint64
 		ExtHash              bc.Hash
 	}
+
+	// Results contains (pointers to) the manifested entries for the
+	// items in body.Results.
+	Results []entry
```

Comment:
If there are restrictions on what possible types can be in this field (not necessarily all `entry` types), it could be nice to include an end-of-line comment about it. (Doesn't have to be verbose or elaborate.)

```
Results []entry // output or retirement
```

Same goes for the other places.